### DISCUSSION TITLE
Welcome to Joystick_ESP32S2 Discussions!

### DISCUSSION THREAD

**Discussion by schnoog** [schnoog/Joystick_ESP32S2#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gokmenhakan** [comment]
Hi, firstly thanks for library. 
How can I find this joystick test software : https://github.com/schnoog/Joystick_ESP32S2/blob/main/ItWorks.jpg
Its name is MultiJoystick Tester 1.2.0.0

**Comment by schnoog** [comment]
Hi,
that should be the one I used
https://github.com/EDDiscovery/MultiJoyStickTest/releases/tag/Release_1_2_0

**Reply by gokmenhakan** [reply]
Thank You Volker.

**Comment by MartinNohr** [comment]
I'm getting an error message when I try to compile any of the examples.
c:\Users\marti\OneDrive\Documents\Arduino\libraries\Joystick_ESP32S2\src/Joystick_ESP32S2.h:51:1: error: expected class-name before '{' token
   51 | {
      | ^

Am I missing a library or something like that?
Thanks.
